src/app/core/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,5 +1,13 @@
 <div class="breadcrumb-container">
-  <ul>
-    <li>Courses</li>
-  </ul>
+  <span *ngFor="let crumb of breadCrumbs; let last = last">
```

Comment:
It would be better to use `ul` and `li` tags here.
In general, if you have a list in your app, you should almost always use HTML tags `ul` (`ol`) and `li`.